### PR TITLE
support TLS (https)

### PR DESCRIPTION
**Pull request by richm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ A [prometheus](http://prometheus.io/) exporter for [rsyslog](http://rsyslog.com)
 ## Rsyslog Configuration
 Configure rsyslog to push JSON formatted stats via omprog:
 ```
+module(load="omprog")
```
Comment (matthiaslee) [inline README.md:8]: 👍 I was going to suggest the same edit

**Review by portante — APPROVED** [review]
Can we consider merging this?

**Comment by danwalkeruk** [comment]
@portante @matthiaslee

Any chance this will ever be merged?

**Comment by portante** [comment]
Besides the fact that it is over 3 1/2 years old, what has prevented this from merging?

**Comment by danwalkeruk** [comment]
I'm not sure, but would be interested to know. 

The most recent commits appear to be from @matthiasr so possibly if he's still involved in this project he could review? Tagging for visibility.

**Comment by matthiasr** [comment]
Sorry, this fork was never meant to be a supported project. I had to do some changes for the use case at SoundCloud, but had no capacity to maintain it as a going fork. Others did, and their effort is now part of prometheus-community: https://github.com/prometheus-community/rsyslog_exporter

If this PR is still valid, please re-submit it against that fork?

**Comment by richm** [comment]
I've been out of logging development for quite some time - would appreciate it if someone else who is interested would take over this PR

**Comment by matthiasr** [comment]
It's also worth noting that the [exporter toolkit](https://github.com/prometheus/exporter-toolkit) now exists and supports TLS in a way that is consistent across exporters, so it may be worth using that.

**Pull request closed without merging**